core/math/bsp_tree.cpp

```diff
@@ -577,12 +577,11 @@ BSP_Tree::BSP_Tree(const PoolVector<Face3> &p_faces, real_t p_error_radius) {
 	error_radius = p_error_radius;
 }
 
-BSP_Tree::BSP_Tree(const Vector<Node> &p_nodes, const Vector<Plane> &p_planes, const Rect3 &p_aabb, real_t p_error_radius) {
-
-	nodes = p_nodes;
-	planes = p_planes;
-	aabb = p_aabb;
-	error_radius = p_error_radius;
+BSP_Tree::BSP_Tree(const Vector<Node> &p_nodes, const Vector<Plane> &p_planes, const Rect3 &p_aabb, real_t p_error_radius)
```

Comment:
I'm not sure about `BreakConstructorInitializersBeforeComma`, that looks a bit hackish. I think we can just go with the format you currently used - once we start using a more recent clang-format, we could use `BreakConstructorInitializers`. I can't find the clang 5.0 documentation online anymore (they are in the process of staging it for the stable release in 2 weeks), but i expect that this option will be available in 5.0.